modules/installation-upi-machines-iso.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-upi-machines-iso-{context}"]
+= Creating {op-system-first} machines using an ISO image
+
+Before you install a user provisioned infrastructure (UPI) cluster on bare
+metal, you need to create {op-system} machines for it to use. You can use an
+ISO image to create the machines.
+
+.Prerequisites
+
+* Obtain the Ignition config files for your cluster.
+* Have access to an HTTP server that you can access from your computer and that
+the machines that you create can access.
+
+.Procedure
+
+////
+This will work for GA.
+. Obtain the {op-system} images from the
+link:https://access.redhat.com/downloads[Product Downloads] page on the Red
+Hat customer portal.
+////
+
+. Upload the control plane, compute, and bootstrap Ignition config files that the
```

Comment:
Is this intentionally keeping at a high level of just using the OpenShift Ignition configurations and leaving customization information for other documentation?

Reply:
Yes, this is intentionally just to support using the OpenShift Ignition configs.